python/lsst/ts/standardscripts/auxtel/daytime_checkout/telescope_and_dome_checkout.py

```diff
@@ -0,0 +1,176 @@
+    Telescope and Dome to ensure it is ready to be released
+    for nighttime operations
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+    **Checkpoints**
+
+    - "Slewing Telecope without tracking...": Before slewing telescope, no dome
+    motion
+    - "Starting tracking test...": Starting sidereal tracking test...
+    - "Slewing Dome...": Before slewing the dome, no telescope motion.
+    - "Parking Telescope and Dome...": Before parking the Telescope and Dome at
+    end of tests. Final Checkpoint.
+
+    **Details**
+
+    This script will perform daytime checkout of Telescope and Dome. It starts
+    by performing a slew of the Telescope without Dome movement. Then sidereal
+    tracking for the Telescope is enabled and the system is left to track for a
+    few minutes. Tracking is disabled and then the Dome is commanded to move to
+    a new position, without Telescope movement. Finally, the Telescope and
+    Dome are returned to a park position and left enabled.
+    """
+
+    def __init__(self, index=1):
+
+        super().__init__(
+            index=index,
+            descr="Execute daytime checkout of AT Telescope and Dome.",
+        )
+
+        # Instantiate atcs and latiss. We need to do this after the call to
+        # super().__init__() above. We can also pass in the script domain and
+        # logger to both classes so log messages generated internally are
+        # published to the efd.
+        self.atcs = ATCS(domain=self.domain, log=self.log)
+
+    @classmethod
+    def get_schema(cls):
+        return None
+
+    async def configure(self, config):
+        # This script does not require any configuration
+        pass
+
+    def set_metadata(self, metadata):
+        """Set estimated duration of the script."""
+
+        metadata.duration = (
+            10.0 * 60
+        )  # Approximate 10min to completion, should depend on what used
+
+    async def run(self):
+
+        await self.assert_feasibility()
+
+        # Disable dome following for initial test
+        await self.atcs.disable_dome_following()
+
+        await self.checkpoint("Slewing Telecope without tracking...")
+
+        # Test point az-el
+        start_az = 0.0
+        start_el = 80.0
+        start_rot = 0
+        await self.atcs.point_azel(az=start_az, el=start_el, rot_tel=start_rot)
+
+        # Stop tracking
+        await self.atcs.stop_tracking()
+
+        await self.checkpoint("Starting tracking test...")
+
+        # Test sidereal-tracking
+        coord = self.atcs.radec_from_azel(az=start_az + 5, el=start_el - 5)
+        await self.atcs.slew_icrs(
+            coord.ra,
+            coord.dec,
+            rot=start_rot,
+            stop_before_slew=False,
+            rot_type=RotType.PhysicalSky,
+        )
+
+        # Sleep for 2 minutes to monitor sidereal tracking
+        await asyncio.sleep(2.0 * 60.0)
```

Comment:
Instead of sleeping, use 

```
            await self.atcs.check_tracking(120.0)
```

This monitors the tracking for the provided number of seconds and fails if something bad happens.

Reply:
Change adopted, also applied to sleep calls in slew_and_take_image_checkout.py